bin/migrate-oats-data/applications/submissions/sql/parcels/primary_contact/application_submission_primary_contact.sql

```diff
@@ -0,0 +1,6 @@
+SELECT apps."uuid" AS submission_uuid,
+    appo."uuid" AS owner_uuid,
+    apps.file_number::bigint
+FROM oats.oats_alr_application_parties oaap
+    JOIN alcs.application_submission apps ON apps.file_number::bigint = oaap.alr_application_id
+    JOIN alcs.application_owner appo ON appo.oats_application_party_id = oaap.alr_application_party_id
```

Comment:
Nitpick: I'm getting a nondescript no entry sign on this file. I think it may be due to a missing semicolon.